zebra-network/src/peer_set/set/tests/prop.rs

```diff
@@ -273,7 +275,8 @@ proptest! {
             }
 
             // this will panic as expected
-            let _ = peer_set.route_broadcast(Request::AdvertiseBlock(block_hash));
+            let response_future = peer_set.route_broadcast(Request::AdvertiseBlock(block_hash));
+            std::mem::drop(response_future);
```

Comment:
Are these explicit `drop`s part of the bug or in response to new lints?

Reply:
In response to new lints, see the commit description of 98fee6cbc53c8ac0e5d72087035d29aba65d8755